app/models/course/assessment/answer.rb

```diff
@@ -2,7 +2,7 @@
 class Course::Assessment::Answer < ActiveRecord::Base
   include Workflow
   actable
-  acts_as :discussion_topic, class_name: Course::Discussion::Topic.name
+  acts_as_discussion_topic display_global: true
```

Comment:
If that is a concern, then the next best is probably `display_globally`, then for the scope it can be `scope :displayed_globally` as you suggested. I think that works as well! 👍